test/unit/verify/test_models.py

```diff
@@ -86,3 +87,13 @@ def test_verification_materials_offline_no_checkpoint(self, signing_bundle):
             InvalidMaterials, match="expected checkpoint in inclusion proof"
         ):
             signing_bundle("bundle_no_checkpoint.txt", offline=True)
+
+    def test_verification_materials_to_bundle_round_trip(self, asset, signing_bundle):
```

Comment:
Thanks! Mind also adding a testcase for the `InvalidMaterials` case above? It should be "just" a matter of loading `VerificationMaterials` from a `.crt` and `.sig` instead and then not explicitly loading in the Rekor entry.

Reply:
Can do!